apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -40,6 +41,7 @@ const styles = {
     padding: 10,
     fontSize: 14,
     fontFamily: '"Gotham 5r", sans-serif',
+    color: color.cyan,
```

Comment:
I think since you replaced all the styles the whole `styles` object defined in this file isn't used anymore and can be deleted.

Reply:
Fixed, thanks!